src/config/colors.toml

```diff
@@ -8,6 +8,13 @@
 
 "commit_id" = "blue"
 "change_id" = "magenta"
+"commit_id prefix" = { bold = true}
+"commit_id rest" = "bright black"
+"change_id prefix" = { bold = true}
+"change_id rest" = "bright black"
+"divergent change_id rest" = "red"
+"divergent change_id prefix" = {fg = "red", bold=true, underlined=true}
```

Comment:
nit: the different attributes of the style don't override each other, and an inner match overrides an outer match, so you should be able to do this:

```
"prefix" = { bold = true}
"rest" = "bright black"
"divergent change_id prefix" = {fg = "red", underlined=true}
"divergent change_id rest" = "red"
```

Reply:
It works, that's pretty neat. I added a comment since it becomes less clear which colors these are.

Reply:
I also ended up relying on the fact that for "divergent change_id rest", a "divergent rest" rule trumps the "change_id" rule, which might not be super-intuitive.

Reply:
Actually, it might be clearer this way, so that for "divergent change_id prefix" different rules don't override the initial red to bright black and then back to red.
```
"prefix" = { bold = true }
"commit_id rest" = "bright black"
"change_id rest" = "bright black"
"divergent prefix" = { underlined = true }
```

Reply:
So, "divergent change_id" overrides "change_id rest". That is not at all clear to me, but it seems to work.

Reply:
Hmm, good question. I don't see why that would work. I would add a `"divergent change_id rest" = "red"` to override it back to red.